src/lib/holocene/keyboard-shortcut/arrow-down.svelte

```diff
@@ -0,0 +1,11 @@
+<svg
+  class="h-4 w-4"
+  aria-hidden="true"
+  fill="currentColor"
+  xmlns="http://www.w3.org/2000/svg"
+  viewBox="0 0 320 512"
+  ><path
+    d="M310.6 246.6l-127.1 128C176.4 380.9 168.2 384 160 384s-16.38-3.125-22.63-9.375l-127.1-128C.2244 237.5-2.516 223.7 2.438 211.8S19.07 192 32 192h255.1c12.94 0 24.62 7.781 29.58 19.75S319.8 237.5 310.6 246.6z"
+  /></svg
+>
```

Comment:
I think we should add these to our `Icon` component, then this component just becomes

```
<Icon name="keyboard-arrow-down" />
<span class="sr-only">Arrow key down</span>
```
Probably not super likely we need these keyboard arrow icons outside of this context (e.g. without the label), but we might.

Reply:
Yup I should do that.